app/utils/url.js

```diff
@@ -14,7 +14,7 @@ export function isValidUrl(url = '') {
 }
 
 export function stripTrailingSlashes(url = '') {
-    return url.trim().replace(/\/+$/, '');
+    return url.replace(/ /g, '').replace(/^\/+/, '').replace(/\/+$/, '');
```

Comment:
I'm not a fan of this, if anyone has a better solution for this please let me know so I can go update it

Reply:
Initially I thought on your first `replace`, you could just simply use the `trim()` function to remove whitespaces on both ends but your implementation works even if there are space/s in between like `"  https://mattermost.com/  /"`.
With that, I'm good with your implementation.
Maybe just add test cases, so this will not break in the future in case needs to modify 😄